backend/src/main/java/fk/prof/backend/verticles/http/HttpHelper.java

```diff
@@ -0,0 +1,25 @@
+package fk.prof.backend.verticles.http;
+
+import fk.prof.backend.exception.HttpFailure;
+import io.netty.handler.codec.http.HttpResponseStatus;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.RoutingContext;
+
+public class HttpHelper {
+  public static void handleFailure(RoutingContext context, HttpFailure exception) {
```

Comment:
I observed some things aren't unit or integration tested. This seems like one of those. We should do a comb-through to identify areas that are untested and identify the best way to test-cover them.